dataplane/forwarding/infra/fwdpacket/packet.go

```diff
@@ -42,6 +42,10 @@ const (
 	LogDesc                // sets the packet's description (prefixed to all messages)
 )
 
+const (
```

Comment:
this isn't right way to turn off the the trace, it's being turned on here https://github.com/openconfig/lemming/blob/main/dataplane/forwarding/fwdport/ports/kernel.go#L110

Reply:
Oops removed. Added common debug package to put the debug flags.